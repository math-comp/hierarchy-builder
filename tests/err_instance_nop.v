From HB Require Import structures.

HB.mixin Record M T := {}.
HB.structure Definition S := { x of M x }.
HB.instance Definition _ : M nat := M.Build nat.
HB.instance Definition _ : M nat := M.Build nat.