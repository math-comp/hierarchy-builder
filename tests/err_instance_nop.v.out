Toplevel input, character 0:
> HB.instance Definition _ : M nat := M.Build nat.
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Warning: HB: no new instance is generated
[HB.no-new-instance,HB,elpi,default]
